src/data/downloads.tsx

```diff
@@ -275,11 +276,11 @@ sudo apt install jellyfin`}
   {
     id: 'portable',
     name: 'Portable',
-    osTypes: [OsType.Linux, OsType.MacOS, OsType.Windows],
+    osTypes: [OsType.DotNet],
     status: DownloadStatus.Official,
     features: [],
     platforms: [Platform.DotNet],
-    description: 'The portable version can be run on any system with a .NET Core runtime.',
+    description: 'The portable version can be run on any system with a .NET runtime.',
```

Comment:
Good idea, though I think it should be even more explicit:

+description: 'The .NET portable version can be run on any system with a .NET runtime using `dotnet jellyfin.dll`.',

Reply:
Well it seems like we cannot use markdown in this description, I change this to 'The .NET portable version can be run on any system with a .NET runtime using jellyfin.dll', which also explicitly states the user has to use the dll directly for this version.